server/plugins/api/Slack.ts

```diff
@@ -0,0 +1,27 @@
+import { APIReturn } from "./APIReturn";
+import { GenericNotifier } from "./GenericNotifier";
+import { GenericConfig } from "./GenericConfig";
+
+interface Config extends GenericConfig {
+	channel?: string;
+	channels?: string[];
```

Comment:
why not instead of both `channel?` & `channels?` just do `channels: string[]` (merge both and drop the `?`)